src/sqlfluff/core/linter/linted_file.py

```diff
@@ -343,12 +343,6 @@ def fix_string(self) -> Tuple[Any, bool]:
                 linter_logger.info(
                     "      - Skipping patch over templated section: %s", enriched_patch
                 )
-            # If we span more than two slices then we should just skip it. Too Hard.
-            elif len(local_raw_slices) > 2:  # pragma: no cover
-                linter_logger.info(
-                    "      - Skipping patch over more than two raw slices: %s",
-                    enriched_patch,
-                )
```

Comment:
My favorite kind of fix -- removing code that didn't do something because was too hard "back in the day". I think the `JinjaTracer` and related work has turned applying tricky fixes from an art into a science.

TBH, I thought I'd already removed all the code like this (it must've been similar code in a different area of the code).